### ISSUE TITLE
Tuning hyper parameters with optimization metric being LogLoss defined explicitly gave bad results

### ISSUE THREAD

**Issue by NadeemNicoR** [pycaret/pycaret#1611]
**Describe the bug**

**To Reproduce**

```python
cannot paste the code as there are some features information that are confidential to the firm
```

**Expected behavior**

**Additional context**

**Versions** pycaret 2.3.3
<details>

</details>

**Comment by NadeemNicoR** [comment]
i have added some notes to the expected behavior not sure why it does not show up above hence i'm adding it again here:

log loss should be minimized, the current condition the function ended up maximizing it, since most of the performance metrics are considered good when their values are max, may be the source code aims at maximizing the metric value for log loss as well

**Comment by NadeemNicoR** [comment]
i have added some notes to the expected behavior not sure why it does not show up above hence i'm adding it again here:

log loss should be minimized, the current condition the function ended up maximizing it, since most of the performance metrics are considered good when their values are max, may be the source code aims at maximizing the metric value for log loss as well

**Comment by Yard1** [comment]
How are you adding the metric?

**Comment by NadeemNicoR** [comment]
add_metric('logloss', 'LogLoss', log_loss, greater_is_better=False, target="pred_proba")

**Comment by moezali1** [comment]
@NadeemNicoR Not sure if this is the bug. Can you try using sklearn directly and see if you get the same behavior?

**Comment by NadeemNicoR** [comment]
@moezali1  when I tried tuning the hyperparameters w.r.t inbuilt ones like AUC i was able to get better cross validation score than the default model but when I did it w.r.t add matric it was giving contrary results. none the less, now I see it is working fine, not sure what had happened before. i tried repeating the experiment for the same sessions id / seed now it is giving results as it ideally should thanks for your help, ill close this thread.
